jj.kak

```diff
@@ -135,6 +136,11 @@ define-command -override jj -params 1.. \
             show_jj_cmd_output log "$@"
         }
 
+        jj_status() {
+            filetype=git-status
+            show_jj_cmd_output status "$@"
```

Comment:
Thanks!

I think we want to create a separate highlighter eventually.
But this repo does not need to be super stable, so I guess we can take it
for now, and maybe remove it later.

Also I realized we might wanna show abbreviations in the docstring (but not in completions I guess).

---

I'm eager to upstream this script; split+squash are quite useful but perhaps
they have too much magic in their current form (reliance on the default log
template is also wrong).